frontend/src/fixtures/commonsFixtures.js

```diff
@@ -14,6 +14,7 @@ const commonsFixtures = {
             "showLeaderboard": true,
             "carryingCapacity": 100,
             "capacityPerUser": 105,
+            "effectiveCapacity": 1050,
```

Comment:
50 players times 105 should be 5250, not 1050.  

All of the other effective capacity numbers in your fixtures seem off.

Please fix this (all of them.)  It will be confusing to future developers to have inconsistent data in the fixtures.